examples/diagram-fit-on-load/README.md

```diff
@@ -28,5 +28,8 @@ const bpmnVisualization = new BpmnVisualization(window.document.getElementById(c
 
 const bpmnContent = ``; // your BPMN 2.0 XML content
 const fitTypeValue = 'Center'; // From radio button or select
-bpmnVisualization.load(bpmnContent, { fitType: FitType[fitTypeValue] });
+const fitMargin = 10; // From input or other
```

Comment:
:warning: the actual example is not using fit margin

Reply:
Yes, but we don't have example for the loading with margin. And in my opinion, as we don't have an API documenttation, we need to explain to the user all the possibilities / paramters of this function 🙂

Reply:
documentation is available with TS declaration file that should be usable by IDE. As we didn't really tested it on a JS project, let's keep the margin part here for now